app/Rest/Resources/PostResource.php

```
@@ -18,6 +19,20 @@ class PostResource extends Resource
      */
     public static $model = Post::class;
 
+    /**
+     * Override the base query to filter posts by visibility for the current user.
+     */
+    public function query(RestRequest $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)